ansible/roles/kafka-logging-setup/files/kafka-lv-brokers.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "kafka-local-brokers"
+  namespace: "local-storage"
+spec:
+  nodeSelector:
+    nodeSelectorTerms:
+    - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - storage001-r640
```

Comment:
Let's plan to take these values as input somewhere maybe in group_vars and template it out. 
1. This is hardcoded
2. We have other components also using local-storage so let's come up with a way where all the components can take advantage of this.

Reply:
I concur, but to take this one step further we should probably have app configuration separate from storage configuration. I will vote kafka playbooks only configure kafka and not local-storage "things".